web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js

```diff
@@ -26,4 +28,4 @@ $(window).mousemove(
 turkey.register_auditor('mouse_movement_specific',
                           auditor_mouse_movement_specific
                             .submit_callable
-                            .bind(auditor_mouse_movement_specific));
+                            .bind(auditor_mouse_movement_specific));
```

Comment:
Nit: Stylistically you want a newline at the end of a file, or we're going to get a spurious "changed line" whenever you add to this.